src/sidebar/components/test/NotebookResultCount-test.js

```diff
@@ -131,7 +131,7 @@ describe('NotebookResultCount', () => {
 
       const wrapper = createComponent({ isLoading: true, resultCount: 2 });
 
-      assert.equal(wrapper.text(), '(2 annotations)');
+      assert.include(wrapper.text(), '(2 annotations)');
```

Comment:
This arises because `Spinner` is no longer mocked here (it being an external component) and `wrapper.text()` has more in it than before.